targets/zephyr/Makefile.zephyr

```diff
@@ -111,6 +131,9 @@ BUILD_CONFIG = O="$(OUTPUT)/zephyr" V=$(V) USER_LIBS="$(LIBS)" USER_LIB_INCLUDE_
 .PHONY: all
 all: jerry zephyr
 
+$(DOTCONFIG):
```

Comment:
Shouldn't this recipe be called `initconfig`, just like the other similar recipes below (e.g., `showconfig`, `mrproper`, `flash`, etc.)? The file itself (i.e., `$(DOTCONFIG)`) is not a dependency anywhere anyway, as far as I can telll.